.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: node_js
 node_js:
-  - 0.8
-  - 0.10
-notifications:
-  email:
-    on_success: change
-    on_failure: change
+  - "0.8"
+  - "0.10"
+  - iojs
+sudo: false # use container based environment
+# notifications:
```

Comment:
Because `change` isn't an email and it shouldn't work? Or am I missing something here?